File: exampleApp/src/main/java/it/trade/android/exampleapp/OrdersStatusActivity.java
package it.trade.android.exampleapp;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.text.method.ScrollingMovementMethod;
import android.widget.TextView;

import java.util.List;

import it.trade.android.sdk.model.orderstatus.TradeItOrderStatusParcelable;

import static it.trade.android.exampleapp.MainActivity.ORDERS_STATUS_PARAMETER;

public class OrdersStatusActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_orders_status);
        TextView textView = (TextView) this.findViewById(R.id.orders_status_textview);
        textView.setMovementMethod(new ScrollingMovementMethod());
        Intent intent = getIntent();
        List<TradeItOrderStatusParcelable> ordersStatusDetailsList = intent.getParcelableArrayListExtra(MainActivity.ORDERS_STATUS_PARAMETER);
        textView.setText(ordersStatusDetailsList.toString());

    }
}

File: exampleAppKotlin/src/main/kotlin/it/trade/android/exampleapp/OrdersStatusActivity.kt
package it.trade.android.exampleapp

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.text.method.ScrollingMovementMethod
import android.widget.TextView

import it.trade.android.sdk.model.orderstatus.TradeItOrderStatusParcelable

import it.trade.android.exampleapp.MainActivity.Companion.ORDERS_STATUS_PARAMETER

class OrdersStatusActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_orders_status)
        val textView = this.findViewById(R.id.orders_status_textview) as TextView?
        textView!!.movementMethod = ScrollingMovementMethod()
        val intent = intent
        val ordersStatusDetailsList = intent.getParcelableArrayListExtra<TradeItOrderStatusParcelable>(MainActivity.ORDERS_STATUS_PARAMETER)
        textView.text = ordersStatusDetailsList.toString()

    }
}
